BO4E-dotnet/COM/Marktrolle.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace BO4E.COM
+{
+    /// <summary>
+    /// Marktrolle 
+    /// </summary>
+    public class Marktrolle : COM
+    {
+        /// <summary>
+        /// rollencodenummer von Marktrolle
+        /// </summary>
+        [JsonProperty(Required = Required.Default)]
+        public string rollencodenummer;
+
+        /// <summary>
+        /// List of Marktrolle. Details siehe <see cref="ENUM.Marktrolle"/>
+        /// </summary>
+        [JsonProperty(Required = Required.Always)] 
+        public ENUM.Marktrolle marktrolle;
+
+        [JsonProperty(Required = Required.Default)]
```

Comment:
Die drei Properties gehören in ein anderes Objekt, oder? Warum sollte die Gasqualität einer Marktrolle zugeordnet sein. Kannst das bitte noch mal mit Flo/Matthias validieren?

Reply:
Ist das Richtig?
![grafik](https://user-images.githubusercontent.com/52197378/75436883-8fed3080-5955-11ea-8627-c9fc27f8c600.png)

Reply:
Nee, das ist uns durchgerutscht. Spannungsebene bis Verlustfaktor ist direkt auf der Messlokation